extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -87,6 +87,28 @@ export const Settings = ({
   const showSlippage = (isPathPayment || isSwap) && !isSoroswap;
 
   async function goToReview() {
+    if (isSoroswap) {
```

Comment:
just moving this up to make sure we go down this path first and return if it's a soroswap